snap/snapcraft.yaml

```diff
@@ -68,11 +69,16 @@ parts:
     source: https://github.com/kk7ds/chirp.git
     override-pull: |
       snapcraftctl pull
-      # invalid snap version "chirp-daily-20220219+git1.3e607109": cannot be longer than 32 characters (got: 34)
+      # snap version cannot be longer than 32 characters
       LATEST_COMMIT_DATE=$(git log -1 --date=format:"%Y%m%d" --format="%ad")
       LATEST_COMMIT_SHORT=$(git rev-parse --short HEAD)
-      version="chirp-daily-${LATEST_COMMIT_DATE}+${LATEST_COMMIT_SHORT}"
-      snapcraftctl set-version "$version"
+      CHIRP_VERSION="daily-${LATEST_COMMIT_DATE}+${LATEST_COMMIT_SHORT}"
```

Comment:
I didn't really pay attention to this earlier, but this will break the automatic update checking routines. I'd really like to figure out a better way to hook the building of stuff like this from the main build system so that the versions line up, if possible.

Reply:
After a few days of thinking on it, I think I got it figured out.
1. I'll set the in app version back to YYYYMMDD of the build date.
2. I'll change the github action that syncs the repo from hourly to on demand.
3. I'll email you a github secret token and github.com url you can use to trigger the sync, which will then trigger the snap build.

I think this should produce matching YYYMMDD versions.